src/forms/datalist/DataList.tsx

```diff
@@ -106,6 +106,7 @@ type DataListProps = {
     spellCheck?: boolean;
     debounce?: boolean; // decide if debounce is needed or not
     debounceTime?: number; // debounce time in miliseconds
+    value?: any;
```

Comment:
Please add documentation for `value` prop

Reply:
Done.